File: MovieCatalogQw/app/src/main/java/com/unlistedi/moviecatalogqw/ui/tvshows/TvShowsViewModel.java
package com.unlistedi.moviecatalogqw.ui.tvshows;

import androidx.lifecycle.ViewModel;

import com.unlistedi.moviecatalogqw.data.MovieEntity;
import com.unlistedi.moviecatalogqw.utils.DataGenerator;

import java.util.ArrayList;

public class TvShowsViewModel extends ViewModel {
    private ArrayList<MovieEntity> tvShows = DataGenerator.getTvShows();

    public ArrayList<MovieEntity> getTvShows() {
        return tvShows;
    }
}

File: MovieCatalogueKu/app/src/main/java/com/unlistedi/moviecatalogueku/ui/tvshows/TvShowsViewModel.kt
package com.unlistedi.moviecatalogueku.ui.tvshows

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import com.unlistedi.moviecatalogueku.data.MovieEntity
import com.unlistedi.moviecatalogueku.utils.DataGenerator

class TvShowsViewModel : ViewModel() {
    val tvShows : ArrayList<MovieEntity> = DataGenerator.getTvShows()
}